src/main/java/org/gradle/profiler/studio/launcher/StudioConfigurationProvider.java

```diff
@@ -20,6 +21,18 @@
 
 public class StudioConfigurationProvider {
 
+    private static final List<String> DEFAULT_JAVA_PATHS = ImmutableList.<String>builder()
+        // MacOS Studio and IntelliJ paths
+        .add("Contents/jre/Contents/Home/bin/java")
```

Comment:
I am not sure if it makes sense to have all Java paths in one list or to have multiple lists, on per OS. WDYT?

Reply:
I split the paths. Even though finding the wrong path for OS is very unlikely, it still might save some weird "gotcha" in the future.